node/Cargo.toml

```diff
@@ -1,22 +1,24 @@
 [package]
 name = "parachain-template-node"
-version = "0.1.0"
-authors = ["Anonymous"]
+authors = [ "Anonymous" ]
```

Comment:
not a concern for this PR, but should we add our names to here? or mention OpenZeppelin? @orangelash